Dockerfile

```diff
@@ -67,13 +57,6 @@ COPY --from=builder /app/we-lang /app/bin/we-lang
 COPY ./bin /app/bin
 COPY ./lib /app/lib
 COPY ./share /app/share
-
-# These files should be mounted by the user at runtime:
-# /root/.wegorc
-# /root/.ip2location.key (optional)
-# /app/airports.dat
-# /app/GeoLite2-City.mmdb
-
```

Comment:
@SuperSandro2000 should these comments be moved to the documentation, in your opinion?

Reply:
I have added a short section in the README. It needs expansion but it is a start.